list2.py

```diff
@@ -15,22 +15,30 @@
 # so [1, 2, 2, 3] returns [1, 2, 3]. You may create a new list or
 # modify the passed in list.
 def remove_adjacent(nums):
-    """Your code goes here.  Edit this docstring."""
-    return
+    return list( set( nums ) )
```

Comment:
FYI 'set' is not an adjacent remove.  It removes ALL repeated occurrences to build a collection of unique items.  
Here's an example test case for this problem:
```
[1, 2, 2, 3, 4, 5, 2, 2, 6] -> [1, 2, 3, 4, 5, 2, 6]
```